specification/operationsmanagement/resource-manager/Microsoft.OperationsManagement/2015-11-01-preview/OperationsManagement.json

```diff
@@ -0,0 +1,422 @@
+                },
+                "provisioningState": {
+                    "type": "string",
+                    "description": "The provisioning state for the solution.",
+                    "readOnly": true
+                },
+                "containedResources": {
+                    "type": "array",
+                    "items":{
+                      "type":"string",
+                      "description":"the azure resourceId of the resource."
+                    },
+                    "description": "The azure resources that will be contained within the solutions. They will be locked and gets deleted automatically when the solution is deleted."                    
+                },
+                "referencedResources": {
+                    "type": "array",
+                    "items":{
+                      "type":"string",
+                      "description":"the azure resourceId of the resource."
+                    },
+                    "description": "The resources that will be referenced from this solution. Deleting any of those solution out of band will break the solution."                    
+                }
+            },
+            "required": [
+                "workspaceResourceId"
+            ],
+            "x-ms-azure-resource": true
+        },
+        "SolutionPropertiesList":{
+          "properties":{
+            "value":{
+              "type":"array",
+              "items":{
+                "$ref":"#/definitions/Solution"
+              },
+              "description":"List of solution properites within the subscription."
+            }
+          },
+          "description":"the list of solution response"
+        },
+        "SolutionPlan":{
+            "description": "Plan for solution object supported by the OperationsManagement resource provider.",
+            "properties":{
+                "name":{
+                  "type":"string",
+                  "description":"name of the solution to be created. For Microsoft published solution it should be in the format of solutionType(workspaceName). SolutionType part is case sensitive. For third party solution, it can be anything."
+                },
+                "publisher": {
+                  "type":"string",
+                  "description": "Publisher name. For gallery solution, it is Microsoft."
+                },
+                "promotionCode":{
+                  "type":"string",
+                  "description":"promotionCode, Not really used now, can you left as empty"
+                },
+                "product":{
+                  "type":"string",
+                  "description":"name of the solution to enabled/add. For Microsoft published gallery solution it should be in the format of OMSGallery/<solutionType>. This is case sensitive"
+                }
+            }
+        },
+        "Solution": {
+            "properties": {
```

Comment:
tags property missing

Reply:
We did not implemented tags for this and there is no plan to invest in this API any more since this is on it's path to be deprecated.

Reply:
We did not implemented tags